### ISSUE TITLE
Calendar Names

### ISSUE THREAD

**Issue by alexanderjoseph** [bitfireAT/davdroid#599]
Since the last update all my calendars have the account name instead of the actual calendar name, this making it impossible to distinguish between them.
I am using Zarafa and aCalendar+

**Comment by rfc2822** [comment]
No, the calendar names are received from the server (`displayname` property), not taken from the account.

**Comment by alexanderjoseph** [comment]
This meaning the server is not sending the proper calendar name?
Why did it work before the update of davdroid?

**Comment by untitaker** [comment]
I've seen a commit that makes DavDroid refetch metadata on every sync. Perhaps the calendarname was always borked serverside, but the issue has now only become apparent because DavDroid synced that information now?

**Comment by rfc2822** [comment]
> This meaning the server is not sending the proper calendar name?

Yes.

> Why did it work before the update of davdroid?

It probably didn't.  Please read [Reporting issues](https://github.com/bitfireAT/davdroid/blob/master/CONTRIBUTING.md#reporting-issues) and provide all information mentioned there, including **detailled steps to reproduce** the issue, **client and server software details**, and **[verbose logs](https://github.com/bitfireAT/davdroid/wiki/How-to-view-the-logs)**. 

> Perhaps the calendarname was always borked serverside, but the issue has now only become apparent because DavDroid synced that information now?

This would be a good explanation.

**Comment by untitaker** [comment]
@alexanderjoseph Could you try other CalDAV clients and see if their calendar names are just as bad?

**Comment by alexanderjoseph** [comment]
Hi there,sorry  for  my late  response.
I  tried caldav-sync. With this App you can select if you want to sync calendar names and colors and it works. Only issue is that every time I  create an entry for a calendar on my phone it forgets this setting, also for calendars with the same colours.

**Comment by alexanderjoseph** [comment]
I have  to  correct  my self.  It has nothing  to  do  with  the  colour.

**Comment by alexanderjoseph** [comment]
Here's another thing I have noticed. 
Right after setting up an account in davdroid the calendar names are correct. This changes with the first synchronisation.

**Comment by devvv4ever** [comment]
What's wrong after the sync? DAVdroid gets the calendar color and name from the server on every sync so that it matches the server-side setting.

**Comment by alexanderjoseph** [comment]
After the sync all calendar names on my phone change to the sever account name (not the account name in DAVdroid, if I remember correctly). This doesn't happens when I use caldav-sync. There is a different issue regarding syncing in caldav-sync. That's why I searched and found your app that worked perfect for me until the next to last (I think) update.

**Comment by devvv4ever** [comment]
I understand. You can change the calendar name on the server to a value you want, DAVdroid should sync this name then.

**Comment by devvv4ever** [comment]
Any news on this? Did it work alexanderjoseph?

**Comment by alexanderjoseph** [comment]
No, even if I change the name of the calendar in the Zarafa WebApp the names are not synchronized correctly :-(
Also I was wrong saying that the calendar-names change to the account name. It is actually the display-name of my account.
